packages/storybook/stories/ChatComposite/snippets/ATryChatCompositeContainer.snippet.tsx

```diff
@@ -14,6 +14,7 @@ export type ContainerProps = {
   errorBar?: boolean;
   showParticipants?: boolean;
   showTopic?: boolean;
+  showRichTextEditor?: boolean;
```

Comment:
The doc here is not consistent with the type of boolean | RichTextEditorOptions?

Reply:
For now, let's leave this as boolean, as don't have any options for this.